hasura/migrations/default/1687894707285_filter_out_samples_contribution_count/up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE OR REPLACE VIEW "public"."contribution_count" AS 
+ SELECT p.id AS profile_id,
+    count(*) AS contributions
+   FROM ((contributions c
+     JOIN users u ON ((u.id = c.user_id)))
+     JOIN profiles p ON (((p.address)::text = (u.address)::text))
+     JOIN circles ci ON ((ci.id = c.circle_id))
```

Comment:
i find it weird this is named `ci` rather than just `c`. was this auto-generated?

Reply:
no It's not auto generated. c is used for contributions so I used ci for circle.